### PR TITLE
Feat/color tokens

### LINKED ISSUE #7539: Colour schemes polishing
Change the colours according to final tokens colours in Figma: https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/branch/BlVnxNzUqrZkbhHgezLczh/s2-Mobile?node-id=9797%3A55518&t=XIZhI2OO9YrqtzWu-1

```[tasklist]
### Tasks

### PR DESCRIPTION
**Pull request by PeKne**
The new token-based colour theme system was applied.
Resolve #7539

## Screenshots:

https://user-images.githubusercontent.com/26143964/222202777-1d9eedac-e27e-4488-a38c-ec1ac4843a2c.mov

https://user-images.githubusercontent.com/26143964/222202728-6074cc46-b278-4c9c-82dc-d4ef16922ed3.mov

### CODE REVIEW

File: packages/theme/designColors.json [hunk 1]
```diff
@@ -0,0 +1,1546 @@
...
+                    "description": "",
+                    "type": "color",
+                    "value": "#eeeeeeff",
+                    "blendMode": "normal",
+                    "extensions": {
+                        "org.lukasoppermann.figmaDesignTokens": {
+                            "styleId": "S:3eb1242659c39f8bef9e8e855d31362300ee1541,",
+                            "exportKey": "color"
+                        }
+                    }
```
Comment (juriczech) [inline packages/theme/designColors.json:15]: nit: Is there better way to export this from figma? We're only using value and it seems useless to store large JSON just to get to 1 property out of 5

File: suite-native/atoms/src/TransactionItem.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```
Comment (juriczech) [inline suite-native/atoms/src/TransactionItem.tsx]: This file is unused and deprecated, you can delete it

**Review by Nodonisko — APPROVED** [review]

File: packages/theme/designColors.json (hunk 1 above)
Comment (PeKne) [reply packages/theme/designColors.json:15]: Unfortunately not. After discussion with designers, this is the leanest version of color config they can get.

File: suite-native/atoms/src/TransactionItem.tsx (hunk 2 above)
Comment (PeKne) [reply suite-native/atoms/src/TransactionItem.tsx]: deleted in dc4a16df4

**Pull request merged**